oxen-rust/docs/dev/IntegrateServerCode.md

```
@@ -156,8 +156,14 @@ If you would like to see the API with `curl` on the command line you can run the
 In order to get a valid auth token you can run add a user to the server via
 
 ```shell
-$ ./target/debug/oxen-server add-user --email ox@oxen.ai --name Ox --output user_config.toml
+$ cargo run -p oxen-server -- add-user --email ox@oxen.ai --name Ox --output user_config.toml
```

markdownlint-cli2 (0.20.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)